docker/php/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
fake the yarn command? (`echo true > /usr/local/bin/yarn && chmod +x /usr/local/bin/yarn`)